hashcompute_test.go

```diff
@@ -124,6 +124,34 @@ func TestNilHashCompute(t *testing.T) {
 	}
 }
 
+func TestExtendHashCompute(t *testing.T) {
```

Comment:
https://github.com/corona10/goimagehash/blob/c0c66db3601904ae329b4b7d768183a90daca3b1/hashcompute_test.go#L207C21-L329

Is this test not covered by the above tests? If not, can we integrate it?

Reply:
Hi,

`TestExtendHashCompute()` covers 2 corner cases in `ExtPerceptionHash()` from `hashcompute.go` that are not covered by the other tests.

```go
	if imgSize <= 0 || imgSize&(imgSize-1) != 0 {
		return nil, errors.New("width * height should be power of 2") // <---
	}
```

and

```go
	if imgSize%lenOfUnit == 0 {
		phash = make([]uint64, imgSize/lenOfUnit)
	} else {
		phash = make([]uint64, imgSize/lenOfUnit+1) // <---
	}
```